### PR TITLE
Added fields for tolerations where applicable

### PR DESCRIPTION
**Pull request by olvesh**
Added fields for tolerations throughout.

### CODE REVIEW

File: helm/alertmanager/values.yaml [hunk 1]
```diff
@@ -85,6 +85,13 @@ ingress:
 ##
 nodeSelector: {}
 
+## Tolerations for use with node taints
+## Ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+##
+tolerations: {}
```
Comment (gianrubio) [inline helm/alertmanager/values.yaml]: Please add an toleration example leaving it commented

File: helm/exporter-node/values.yaml [hunk 2]
```diff
@@ -16,6 +16,15 @@ resources:
...
 
 # default rules are in templates/node.rules.yaml
-# prometheusRules: {}
\ No newline at end of file
+# prometheusRules: {}
+
+## Tolerations for use with node taints
+## Ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+##
+tolerations: {}
```
Comment (gianrubio) [inline helm/exporter-node/values.yaml]: this seems duplicate with line 19

File: helm/grafana/values.yaml [hunk 3]
```diff
@@ -1,5 +1,10 @@
 nodeSelector: {}
 
+## Tolerations for use with node taints
+## Ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+##
+tolerations: {}
```
Comment (gianrubio) [inline helm/grafana/values.yaml]: also add a tolerations example

File: helm/prometheus-operator/values.yaml [hunk 4]
```diff
@@ -36,6 +36,11 @@ kubeletService:
 ##
 nodeSelector: {}
 
+## Tolerations for use with node taints
+## Ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+##
+tolerations: {}
```
Comment (gianrubio) [inline helm/prometheus-operator/values.yaml]: also add an example

**Comment by gianrubio** [comment]
There's a conflict on your branch, could you rebase from master and bump all charts versions on https://github.com/olvesh/prometheus-operator/blob/97537bd773d3cbd2487bafce5a0fbe291f808676/helm/kube-prometheus/requirements.yaml

**Comment by olvesh** [comment]
Weird, I just did a rebase, or so I thought..  Should I update requirements.yaml as well? That won't work until they are uploaded anyway I guess?

**Comment by olvesh** [comment]
Should I squash these commits as well? Getting a bit messy now.

**Comment by olvesh** [comment]
I’ll retest this locally later today. Not sure what failed in the e2e-test now.

**Comment by gianrubio** [comment]
The error is in the prometheus-operator chart, related to the toleration.

``` ++pwd
+helm install --namespace=helm-monitoring /home/travis/gopath/src/github.com/coreos/prometheus-operator/helm/kube-prometheus --name kube-prometheus
Error: error validating "": error validating data: [found invalid field operator for v1.PodSpec, expected type array, for field spec.template.spec.tolerations, got string]
make: *** [e2e-helm] Error 1
``` 

Did you have the same error when you [tested](https://github.com/coreos/prometheus-operator/tree/master/helm#how-to-test) in your machine?

**Comment by olvesh** [comment]
I forgot to update the charts folder. My local checks worked, but I had an error in the exporter-node that didn't show up locally. Haven't grokked your dev flow yet :-) 
Think it should work now.

**Comment by gianrubio** [comment]
lgtm! thanks @olvesh

**Pull request merged**